### PR TITLE
Update link in cards and remove unnecessary html comments

### PR DESCRIPTION
**Pull request by sinscary**

### CODE REVIEW

File: plino/templates/index.html [hunk 1]
```diff
@@ -591,20 +591,28 @@ <h2 class="section-subtitle wow fadeInDown" data-wow-duration="1000ms" data-wow-
...
-             <h4>Full Stack Flask, Django and Rails Developer, Web Dev @ <a href="http://srmadt.com/">SRMADT</a></h4>
+             <h4>Full Stack Flask, Django and Rails Developer, Head of Web Dev @ <a href="http://srmadt.com/">SRMADT</a></h4>
              <p>
-             A Full Stack Web Developer and JavaScripter. Mostly work on Django, Flask and Ruby on Rails FrameWorks. Also develops Hybrid Mobile apps using Ionic framework and Angular Js. <br>Also had been Captain of state level Kabaddi Champion Team and loves Cooking in spare time</p>
+             A Full Stack Web Developer and JavaScripter. Expert in Ruby on Rails, Flask and Express.js FrameWorks. Also develops Hybrid Mobile apps using Ionic framework and Angular Js. <br>Also had been Captain of state level Kabaddi Champion Team and loves Cooking in spare time</p>
            </div>
            <div class="card-footer">
              <ul>
                <li>
+                 <a href="https://github.com/sinscary"><i class="fa fa-github"></i></a>
```
Comment (prodicus) [inline plino/templates/index.html]: Removing the github link?

Comment (sinscary) [inline plino/templates/index.html]: No, i changed the order of link, didn't removed it

Comment (prodicus) [inline plino/templates/index.html]: Ok. Merging it

Comment (sinscary) [inline plino/templates/index.html]: :+1:

**Pull request merged**